### ISSUE TITLE
bug(MDCDialog) Uncaught TypeError: e is not a function at Object.i [as createFocusTrapInstance]

### ISSUE THREAD

**Issue by Jelmer7** [material-components/material-components-web#4484]
### What MDC Web Version are you using?

1.0.0

### What browser(s) is this bug affecting?

Chrome version: 72.0.3626.119 (64 bits)

### What OS are you using?

Windows 10

### What are the steps to reproduce the bug?
1. Clone: https://github.com/Jelmer7/mdc-dialog-test
2. Run: npm install
3. Run: npm run build 
4. Open index.html

**or**

1. Install locally following Getting started guide on material.io
    **Using MDC Web with Sass and ES2015**
2. Update packages to 1.0.0
3. Create page with mdc dialog (literally copied from material.io)

### What is the expected behavior?

No errors initilializing dialog

### What is the actual behavior?

Console gives:
Uncaught TypeError: e is not a function...

### Any other information you believe would be useful?

This only happens locally, using CDN this problem does not occur
for demonstration clone: https://github.com/Jelmer7/mdc-dialog-test

**Comment by BhavyaSingh2611** [comment]
Try using unpkg.com

**Comment by kfranqueiro** [comment]
Thanks for reporting this, and sorry for the trouble! I'm a little perplexed at the fact we didn't encounter this problem ourselves, but I can definitely reproduce it in your repo (and the same code works against 0.44), so that was very helpful.

This was caused by us changing our focus-trap imports to `import *`, which we did for the sake of incorporating its typings. If I change to separate `import createFocusTrap` and `import {...}` for the types, then it looks like it works. I'll file a PR soon.
